### DISCUSSION TITLE
Syntax Colors not showing for developer-made classes (starting from V1.3.2)

### DISCUSSION THREAD

**Discussion by PrezSkroob** [xamarin/XamarinCommunityToolkit#1824] [Q&A] [answered]
So, I recently downloaded the latest Xamarin Community Toolkit (V2.0.0) to take a look at the BadgeView class. After I installed it into my copy of Visual Studio 2019 Pro (V16.10), I found that any classes I make do not show up with the same syntax colors  as it did before.

For example, say I had a class called XYZ, and I tried to declare the following:

 XYZ newVar = new XYZ( );

It would not show the usual colors it did before I installed the toolkit. It would all just be in black text.

When I uninstalled it, I was able to see the color scheme as it did before (in my case, I would have a sky blue color for XYZ).

So, I started to downgrade the XCT down until I got to V1.3.1, at which point the syntax highlighting returned. It seems that everything after that version (V1.3.2 and V2.0.0) does not show the same behaviour.

Is there perhaps a setting somewhere I have to change? Or could this be a known issue? (I haven't been able to dig up anything on it, so my apologies if it's already been brought up)

**Comment by pictos** [comment, marked as answer]
@PrezSkroob does this sounds familiar?
https://github.com/xamarin/XamarinCommunityToolkit/issues/1815

**Reply by PrezSkroob** [reply]
Yep, it certainly does. It looks like the same issue I'm experiencing.
